source/command_bus.coffee

```diff
@@ -7,19 +7,22 @@ class Space.messaging.CommandBus extends Space.Object
   }
 
   _handlers: null
+  _onSendHooks: null
 
   constructor: ->
     super
     @_handlers = {}
+    @_onSendHooks = []
 
-  send: (command, callback) ->
+  send: (command, callback, options={}) ->
     if @meteor.isServer
       # ON THE SERVER
       handler = @_handlers[command.typeName()]
-      if not handler?
+      hook(command) for hook in @_onSendHooks when !options.ignoreHooks
+      if !handler? and !options.silent
```

Comment:
No, normally it would fail here if there is no command handler in an app. We can think about removing this "restriction", although it makes sense in a real world app. For testing we don't always want to have to setup a stub command handler just to test if a command was sent.